pkg/jobs/trigger_interval.go

```diff
@@ -0,0 +1,69 @@
+package jobs
+
+import (
+	"errors"
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// IntervalTrigger implements the @interval trigger. It triggers a job on a
+// fixed period of time.
+type IntervalTrigger struct {
+	du   time.Duration
+	in   *TriggerInfos
+	done chan struct{}
+}
+
+// NewIntervalTrigger returns a new instace of IntervalTriven given the
+// specified options.
+func NewIntervalTrigger(infos *TriggerInfos) (*IntervalTrigger, error) {
```

Comment:
To be clear, for the moment, we have no persistence of when the last job was executed, right?

If I have a konnector that is launched each month, but I restart the stack every weeks (for updates maybe), the trigger will never be fired. I think it is something we should address, but it can be later. A todo can be enough for the moment.